src/main/webapp/go.js

```diff
@@ -184,9 +224,18 @@ function handleDestinationAnswer() {
  * Disable lint check because startHunt() is called from go.html.
  */
 function startHunt() { //eslint-disable-line
-  destIndex = 0;
+  hunt.start();
   sendIndexToServlet(0);
   updateToCurrentState(0);
+  startTimer();
```

Comment:
I think the timer could also be moved to the hunt class as well, since when you start the hunt the timer should start.

The you could have methods like hunt.getTimeElapsed() which would return difference which you would use in updateTimer()